libgraph/src/analytics/leiden_clustering/leiden_clustering.cpp

```diff
@@ -28,11 +28,11 @@
 using namespace katana::analytics;
 namespace {
 
-template <typename EdgeWeightType>
+template <typename EdgeWeightType, typename ViewTy>
 struct LeidenClusteringImplementation
     : public katana::analytics::ClusteringImplementationBase<
           katana::TypedPropertyGraphView<
-              katana::PropertyGraphViews::Undirected,
+              ViewTy,
```

Comment:
Can you please remove the graph types as a separate structure similar to what is in Louvain (`struct GraphTypes`) to make it more readable?